agent/tests/test_security.py

```
@@ -526,28 +547,34 @@ class TestBotResolutionSecurity:
 
     def test_bot_names_empty_list_no_bots(self):
+        """Verify bot names empty list no bots."""
         from iconfucius.skills.executor import _resolve_bot_names
         assert _resolve_bot_names({"bot_names": []}) == []
 
     def test_all_bots_false_ignored(self):
+        """Verify all bots false ignored."""
         from iconfucius.skills.executor import _resolve_bot_names
         assert _resolve_bot_names({"all_bots": False}) == []
 
     def test_all_bots_true_resolves_all(self, odin_project):
```

Ruff (0.15.2)

[warning] Unused method argument: `odin_project`

(ARG002)